README.md

```diff
@@ -1,27 +1,508 @@
 # React Changing State
 
-## Objectives
+## Overview
 
-1. Practicing updating state in React
-2. Practice passing state through props
-3. Describe how React rerenders on state changes
+React allows you to structure your application using small, reusable components,
+but React isn't really just about creating static pages, it's about building
+**react**ive user interfaces that enable you to coordinate complex user
+interactions.
 
-## Overview
+In this lesson we will be learning about how you can update the state of
+components. You are going to learn about the difference between `state` and
+`props` and what happens when state changes in your application occur.
+
+## State vs Props
+
+Managing state is hard. React allows you to encapsulate two different "kinds" of
+state: "internal" state that can only be mutated and accessed by the component
+that manages it (`state`) and "external" state that gets passed down to the
```

Comment:
Not sure if we should refer to props as external state, wouldn't that make things slightly confusing?